products/sle/main.pm

```diff
@@ -863,8 +863,8 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/install_package";
         loadtest "virt_autotest/update_package";
         # Skip reset_partition for s390x due to there just be 42Gib disk space for each s390x LPAR
-        loadtest "virt_autotest/reset_partition" if (!is_s390x and get_var('VIRT_PRJ1_GUEST_INSTALL'));
-        loadtest "virt_autotest/reboot_and_wait_up_normal" if (!get_var('AUTOYAST') && get_var('REPO_0_TO_INSTALL'));
+        loadtest "virt_autotest/reset_partition" if (is_x86_64 and get_var('VIRT_PRJ1_GUEST_INSTALL') and !get_var('LTSS'));
+        loadtest "virt_autotest/reboot_and_wait_up_normal" if (check_var('SCC_REGISTER', 'none') && get_var('REPO_0_TO_INSTALL'));
```

Comment:
this SCC_REGISTER check can be replaced by[ is_registered_sles](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/daf38bea50e809da9d62bee7b42e187e0bbd6b8f/tests/virt_autotest/virt_utils.pm#L744)

Reply:
I am afraid `is_registered_sles` cann't be used at here as it is defined in

```
./tests/virt_autotest/virt_utils.pm:sub is_registered_sles {
```

Reply:
I see, it should be better to put it in lib/virt_utils.pm... I guess you would not bother to move it ;-)?

Reply:
Ok, I'll move. but it need some time as it is called in multiple places. I'll have more verification runs when machines are not busy. The related PR(https://github.com/SUSE/qa-automation/pull/851) need not wait for this PR.

Reply:
Thank you.

Reply:
One more question, is it appropriate to add an extra dependency to main.pm?

see my new commit to main.pm:
```
use virt_autotest::utils qw(is_registered_sles)
```

Reply:
ok. Waiting for idle machines to verify.